deploy/template.yaml

```diff
@@ -453,6 +453,7 @@ Resources:
               IDENTITY_ADDRESS_URL: "/perfTest/identity/orange/addressUrl"
               IDENTITY_BUILD_CORE_URL: "/perfTest/identity/build/coreUrl"
               IDENTITY_BUILD_ORCH_STUB_URL: "/perfTest/identity/build/orchStubUrl"
+              IDENTITY_ORANGE_BUILD_OTG_URL: "/perfTest/identity/orange/OTGUrlBuild"
```

Comment:
Please ensure that a parameter store variable is created with this path in the SSM before running the smoke test in the pipeline.